### ISSUE TITLE
Don't warn about base dependencies

### ISSUE THREAD

**Issue by revodavid** [RevolutionAnalytics/checkpoint#154]
Everytime I run a script with a `checkpoint` call I see a message like this:

```
Warning: dependencies ‘utils’, ‘stats’, ‘methods’, ‘grid’ are not available
```

This isn't a helpful message for base packages, which are installed with R. Don't show this message for base packages.

**Comment by andrie** [comment]
Since it's not trivial to suppress only *some* warnings, the fix for this is to suppress *all* warnings when installing packages.
